TypeCobol/Compiler/CupParser/NodeBuilder/ProgramClassBuilder.cs

```diff
@@ -1180,13 +1180,6 @@ public virtual void StartWhenConditionClause(List<TypeCobol.Compiler.CodeElement
                     condition.SelectionObjects = new EvaluateSelectionObject[1];
                     condition.SelectionObjects[0] = new EvaluateSelectionObject();
                     condition.SelectionObjects[0].BooleanComparisonVariable = new BooleanValueOrExpression(whensearch.Condition);
-
-                    var conditionNameConditionOrSwitchStatusCondition = whensearch.Condition as ConditionNameConditionOrSwitchStatusCondition;
```

Comment:
Just for the records: this code was added by #921.
Originaly it was on CobolNodeBuilder.cs class
see f2314c42f8fe9a307d31b33f8af74b3165d070cc